src/main/java/com/microsoft/azure/helium/config/BuildConfig.java

```diff
@@ -12,10 +18,26 @@
 
     public String getBuildVersion(){
 
-        String buildName = buildProperties.getName();
-        String buildVersion = buildProperties.getVersion();
-        String buildTime =  String.valueOf(buildProperties.getTime().getEpochSecond());
-        System.out.println("buildName " + buildName +"_"+ "buildVersion "+ buildVersion + "_" + "buildTime "+ buildTime );
-        return buildVersion+"."+buildTime ;
+        Instant buildTime = buildProperties.getTime();
+
+        DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+                .withZone(ZoneId.systemDefault());
+
+        System.out.println(DATE_TIME_FORMATTER.format(buildTime));
+
+        //major.minor.MMdd.hhmm
+        String major = buildProperties.getVersion();
```

Comment:
Can you combine the two date formats?

Reply:
DateFormatter cannot combine multiple formats in it's parsing , hence this is the best way we can do.
https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html